atel0/AAEmu
# Centralize version information for all projects

## Description


This PR creates a centralized file (Directory.Build.props) for handling props that are common to all projects in the AAEmu solution and that should be kept in sync across projects. 

## Related Issue




N/A

## Motivation and Context


Main motivation is removing the need to make multiple changes to keep version information in sync across all projects. With version information controlled by Directory.Build.props we have an easier way to Up/Down versions, plus it also makes it easier to add some automation to control releasing/publishing.

## How Has This Been Tested?




- Build the solutions
- Check the version info with exiftool
- Run the Game/Login servers

No issues found.

## Screenshots (if appropriate):
N/A

## Types of changes

- [x] Refactor (any changes that deal with code organization, code flow, logic, decoupling, etc.)
- [ ] Documentation Update (self explanatory)
- [ ] Misc Tasks (Repo organization, maintenance and other tasks that don't fit any of the above)

## Checklist:
